pom.xml

```diff
@@ -601,6 +601,11 @@
             <!-- Explicitly override all netty artifacts to get the version we want. Doing netty-all
              doesn't seem to correctly override its individual dependencies, so all are enumerated
              here -->
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-all</artifactId>
+                <version>${netty.version}</version>
+            </dependency>
```

Comment:
Are these needed because of the upgrade?

Reply:
I noticed that `mvn dependency:tree` shows older versions for `netty-all`.

The comment here explains it pretty much: https://github.com/confluentinc/ksql/pull/10081
```
<!--
    Explicitly override all netty artifacts to get the version we want. Doing netty-all
    doesn't seem to correctly override its individual dependencies, so all are enumerated
    here
-->
```

Previously we were only overriding the individual child artifacts, but some other deps transitively pull in `netty-all` for which we did not declare an override.